cmd/osctl/cmd/cluster.go

```diff
@@ -139,24 +134,25 @@ func create(ctx context.Context) (err error) {
 	if inputDir != "" {
 		configBundleOpts = append(configBundleOpts, config.WithExistingConfigs(inputDir))
 	} else {
-		genOptions := []generate.GenOption{
-			generate.WithInstallImage(nodeInstallImage),
-		}
-
-		genOptions = append(genOptions, provisioner.GenOptions()...)
+		var genOptions []generate.GenOption
+		endpointList := []string{}
 
 		if forceEndpoint != "" {
-			genOptions = append(genOptions, generate.WithEndpointList([]string{forceEndpoint}))
+			endpointList = append(endpointList, forceEndpoint)
 			provisionOptions = append(provisionOptions, provision.WithEndpoint(forceEndpoint))
 		} else if forceInitNodeAsEndpoint {
-			genOptions = append(genOptions, generate.WithEndpointList([]string{ips[0].String()}))
+			endpointList = append(endpointList, ips[0].String())
 		}
 
+		endpointList = append(endpointList, "127.0.0.1")
```

Comment:
nit: We should have a comment here prefixed with `NB:` to indicate that it is crucial to set this _last_: https://en.wikipedia.org/wiki/Nota_bene .

Reply:
I wonder if it should be more of `switch`-like logic here? As right now it appends to the list, while it was supposed to be replacing the list depending on the config? 

Once we do client LB, having 127.0.0.1 while it's not running current cluster might lead to fun results.

Reply:
I _think_ we append only for the purposes of getting a valid cert SANs? @rsmitty is that right?

Reply:
fixed in amend.